packages/x-scheduler/src/joy/models/events.ts

```diff
@@ -0,0 +1,20 @@
+import { SchedulerValidDate } from '../../primitives/utils/adapter/types';
+
+export interface CalendarEvent {
+  /**
+   * The unique identifier of the event.
+   */
+  id: string;
+  /**
+   * The title of the event.
+   */
+  title: string;
```

Comment:
Same, I'm not adding any way to use custom props to pass the title / start / end to avoid complexity for now and focus on the features themselves.